tests/e2e/transaction/transactionSubmission.test.ts

```diff
@@ -606,7 +605,7 @@ describe("transaction submission", () => {
         signaturesRequired: 2,
       });
 
-      const authKey = AuthenticationKey.fromPublicKey({ publicKey: multiKey });
```

Comment:
is this function removed? if so, need to mention it on changelog

Reply:
It's still there, just marked as deprecated.
Should I still mention it in the current changelog, or only when we decide to fully remove it?